src/ctapipe/reco/impact.py

```diff
@@ -85,51 +90,59 @@ class ImPACTReconstructor(Component):
 
     """
 
+    minimiser = traits.CaselessStrEnum(
+        ["minuit", "l-BFGS"],
+        default_value="minuit",
+        help="name minimiser to use in the fit",
+    ).tag(config=True)
+
+    use_time_gradient = traits.Bool(
+        default_value=False, help="Use time gradient in ImPACT reconstruction"
```

Comment:
Does this option have any additional requirements on the templates? I.e. do they need to contain time information?